### PR TITLE
Create marcelsiegert-his-new-file.md

### PR DESCRIPTION
**Pull request by marcelsiegert**
I'd love to add this file to your repository.

### CODE REVIEW

File: marcelsiegert-his-new-file.md [hunk 1]
```diff
@@ -0,0 +1 @@
+My new file. :)
```
Comment (spindev) [inline marcelsiegert-his-new-file.md:1]: Maybe better?
```suggestion
This is my new file. :)
```

**Review by spindev — COMMENTED** [review]
One suggestion. Please check @marcelsiegert.

File: marcelsiegert-his-new-file.md (hunk 1 above)
Comment (marcelsiegert) [reply marcelsiegert-his-new-file.md]: Yes, that's a great idea!

**Review by spindev — APPROVED** [review]
LGTM.

**Pull request merged**